### DISCUSSION TITLE
Limit Create

### DISCUSSION THREAD

**Discussion by paingchan** [filamentphp/filament#12576] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

<img width="807" alt="image" src="https://github.com/filamentphp/filament/assets/54994337/dc9fbc39-fd42-4f0d-9618-f4710f0d39ab">
Free plan only can create 5 data
i want to disable create data when inventory have 5 data

**Comment by leandrocfe** [comment]
What is the issue?
